### PR TITLE
Review changelog and documentation changes for 2022.3

### PR DESCRIPTION
**Pull request by seanbudd**
**Must be squash merge**

Comparison command:
`git diff release-2022.2...origin/branchBeta2022.3 -- "**/en/*.t2t" "**/developerGuide.t2t" `

Common mistakes to look for:

- Issue/PR reference is incorrect
- Incorrect spelling.
- Lists not terminated with a final - on a new line, matching indentation
- List items not indented by a multiple of 2 spaces
- Single grave marks ` instead of double grave marks ``
- Shortcuts added without code markdown, use two 'grave' characters (``NVDA+d``)
- Double spaces
- Inconsistent use of single vs double quote

### CODE REVIEW

**Review by Qchristensen — DISMISSED** [review]
Reads well, good work, and great to highlight our community contributors!

**Comment by codeofdusk** [comment]
Wow, looks like NV Access is really picking up the pace with getting releases out the door!

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -4,12 +4,19 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2022.3 =
+A significant amount of this release was contributed by the NVDA development community.
+This includes table say all commands, delayed character descriptions, and improved Windows Console behaviour.
```
Comment (codeofdusk) [inline user_docs/en/changes.t2t:8]:
```suggestion
This includes table say all commands, delayed character descriptions, and improved Windows Console support.

```

**Review by feerrenrut — DISMISSED** [review]

**Review by feerrenrut — APPROVED** [review]

**Comment by XLTechie** [comment]
@seanbudd
>  *  Inconsistent use of single vs double quote.le of 2 spaces

What is a quote.le?

**Comment by seanbudd** [comment]
@XLTechie - copy paste typo

**Pull request merged**